### PR TITLE
KPI data not found

### PR DESCRIPTION
**Pull request by xaun**
@hedudelgado @cesar-tonnoir please review

### CODE REVIEW

File: src/impac-angular.module.js [hunk 1]
```diff
@@ -121,14 +122,18 @@ angular.module('impac.components.dashboard-settings',
...
     'impac.components.widgets-common.editable-title',
     'impac.components.widgets-common.info-panel',
     'impac.components.widgets-common.time-period-info',
     'impac.components.widgets-common.top-buttons',
     'impac.components.widgets-common.autofocus'
   ]
 );
+angular.module('impac.components.common-settings',
+  [
+    'impac.components.common-settings.data-not-found'
```
Comment (cesar-tonnoir) [inline src/impac-angular.module.js]: Can you review the naming? "data not found" state is not a setting per say... maybe just "common" ?

File: src/services/kpis/kpis.svc.coffee [hunk 2]
```diff
@@ -260,7 +260,10 @@ angular
               (response) ->
                 # When no target has been defined
                 if response.data.error && response.data.error.code == 422
-                  # TODO force edit mode
+                  # TODO: this can be removed when Impac! API returns layout config for draft
+                  # mode KPIs. As then KPIs with no kpi.data and kpi.layout could be
+                  # considered in draft mode.
+                  kpi.is_draft = true
```
Comment (cesar-tonnoir) [inline src/services/kpis/kpis.svc.coffee]: `snake_case` in coffeescript?

Comment (xaun) [inline src/services/kpis/kpis.svc.coffee]: Hmm was on automatic here, I think my brain was thinking I was mocking a back-end attribute. But you're right, this should not be snake_case.

**Comment by xaun** [comment]
not ready for merge

**Pull request closed without merging**